client/src/modules/lead/EditLead/EditLeadContent/EditLeadTabs/AddActivity/AddActivity.js

```diff
@@ -124,6 +128,23 @@ export default class AddActivity extends Component {
     return options;
   };
 
+  componentDidMount = () => {
+    if (this.props.activity) {
```

Comment:
you should extract `this.props.activity` as `activity` or smth to make code shorter